<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1510</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,1">
        <item>
         <widget class="QWebEngineView" name="webEngineView">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame_main">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1510</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_GeoJSON_File"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_elements"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionSimulationv1"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSimulation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_GeoJSON_File">
   <property name="text">
    <string>Import GeoJSON File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSimulationv1">
   <property name="text">
    <string>Simulation v1</string>
   </property>
   <property name="toolTip">
    <string>Simulation v1</string>
   </property>
  </action>
  <action name="actionAdd_elements">
   <property name="text">
    <string>Add elements</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>754</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
